test/unit/specs/pages/server/browserpage.spec.js

```diff
@@ -0,0 +1,15 @@
+
+import BrowsePage from '@/pages/server/BrowsePage';
+import render from '../../../test-utils/render';
+
+describe('BrowsePage', () => {
+  it('should render correct contents', () => {
+    const wrapper = render(BrowsePage);
+
+    expect(wrapper.find('.browse-page'));
+    expect(wrapper.find({ name: 'header-section' }).vm).toBeDefined();
+    expect(wrapper.find({ name: 'search-grid-filter' }).vm).toBeUndefined();
```

Comment:
Why include this test? The search-grid-filter is never imported to the BrowserPage component, so this will always pass.